osbuild/util/rhsm.py

```diff
@@ -0,0 +1,54 @@
+"""Red Hat Subscription Manager support module
+
+This module implements utilities that help with interactions
+with the subscriptions attached to the host machine.
+"""
+
+import configparser
+import re
+
+
+class Subscriptions:
+    def __init__(self, sections):
+        self.sections = sections
```

Comment:
If I understand the code correctly, `sections` are actually repositories, no?

Reply:
Yes, do you think it is not appropriate naming?

Reply:
I was just thinking, if they are repositories then that would be a clearer name.